libparsec/crates/types/src/lib.rs

```diff
@@ -123,6 +123,9 @@ pub type IndexInt = u64;
 // pub type IndexInt = std::num::NonZeroU64;
 // pub type IndexOffsetInt = u64;
 
+/// Used to identify client handle
+pub type Handle = u32;
```

Comment:
This type is specific to the implementation provided for the JS Client, I suggest to keep it in `libparsec` crate and not add it to `libparsec_types`